release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -42,6 +42,45 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### Language
 
+##### Module level complex variable support
+
+Tuple variable, Record variable and Error variable declaration is now allowed at module level. Unlike simple variables, complex variable declaration must have an initialization.
+Complex variable declaration cannot contain `isolated` or `configurable` qualifier.
+```ballerina
+// Module level tuple variable declaration
+[int, float] [a, b] = [1, 2.5];
+
+// Module level record variable declaration
+Person {name: Fname, married: Married} = {name: "Jhone", married: true};
+
+type Person record {|
+    string name;
+    boolean married;
+|};
+
+// Module level error variable declaration
+UserDefinedError error(message, basicErrorNo = detail) = getError();
+
+type UserDefinedError error <BasicErrorDetail>;
+type BasicErrorDetail record {|
+    int basicErrorNo?;
+    anydata...;
+|};
```

Comment:
Shall we move these also to before where they are used?

Reply:
removed custom error